apps/examples/testcase/le_tc/kernel/itc_pthread.c

```diff
@@ -81,7 +82,8 @@ static void *thread_setgetname(void *param)
 */
 static void *thread_equal_check(void *param)
 {
-	if (pthread_equal(g_pthread, pthread_self()) == false) {
+	sleep(1);
```

Comment:
It looks strange. Why do we need to sleep before execution without any conditional?

Reply:
This change is no longer required. So, I am reverting it.